src/components/landing/SliderForEvents/SliderForEvents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import PropTypes from "prop-types";
+import Image from "../../../mockup-images/football-g10e9bda93_1920.jpg";
```

Comment:
Where this file exists?

Reply:
What do you mean? This is a temporary file that exists in the folder with mockups